doc/7/controllers/auth/login/index.md

```diff
@@ -9,7 +9,11 @@ description: Authenticate a user
 
 Authenticates a user.
 
-If this action is successful, then the [jwt](/sdk/js/7/core-classes/kuzzle/properties) property of this class instance is set to the new authentication token.
+When the SDK is constructed with the option [cookieAuth](/sdk/js/7/core-classes/kuzzle/constructor)
+- If this action is successful, then http cookie is set to the new authentication token.
+  
+Otherwise
+- If this action is successful, then the [jwt](/sdk/js/7/core-classes/kuzzle/properties) property of this class instance is set to the new authentication token.
```

Comment:
After the use of **Otherwise** I figured the next 'condition' should be that this action was NOT successful right?
```suggestion
- If this action is successful, then http cookie is set to the new authentication token.
  
Otherwise
- The [jwt](/sdk/js/7/core-classes/kuzzle/properties) property of this class instance is set to the new authentication token.
```
Feels like:
If this action is successful...
**Otherwise,**
If the action is sucessful (wot?)

Should be:
If this action is successful...
**Otherwise,**
The jwt property....